src/Components/Todo.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function Todo({ lists, completeTodo }) {
+    const handleClick = id => {
```

Comment:
This `handleClick` should be in the parent component and passed down as a prop. We don't want smaller components to be handling logic unless it is super specific its need. Since the state is in the parent, this should be handled there.